cassandra/generator/photos-generator.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const faker = require('faker');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+
+const { restaurants } = require('../../data/restaurants');
+
+const randomize = (min, max) => {
+  return Math.floor(Math.random() * (max - min) + min)
+}
+
+// specify how many photos per bucket ~
+const getRandomPhotoUrl = () => {
+  const bucketUrl = 'https://toptable-gallery.s3-us-west-1.amazonaws.com/';
+  const photoEndpoint = getRandomIntInclusive(1, 1000).toString();
+  return `${bucketUrl}${photoEndpoint}.jpg`;
+};
+
+// https://hackreactor-restaurant-images.s3-us-west-2.amazonaws.com/newseed/1a.jpg
+// https://hrsf130-tkout-photo-gallery.s3.us-east-2.amazonaws.com/1.png
+// npx bulksplash
+
+const generatePhotos () => {
+  writer.pipe(fs.createWriteStream('photos.csv'));
+  for (var i = 0; i < 100000; i++) {
+    i === 1000
+      ? console.log('o-o   o-o  o--o o---o ')
+        : i === 2000
+        ? console.log('o     o     |       /')
+          : i === 5000
+          ? console.log('|  -o |  -o O-o   -O-
+          ')
+            : i === 8000
+            ? console.log('o   | o   | |     /
```

Comment:
what is going on here with these console logs?  are you using regex?

Reply:
ASCII generator for fun console.log output to show progress in csv generation!